docs/core/entity/media-player.md

```diff
@@ -118,3 +118,25 @@ Optional. What type of media device is this. It will possibly map to google devi
 | tv | Device is a television type device.
 | speaker | Device is speakers or stereo type device.
 | receiver | Device is audio video receiver type device taking audio and outputting to speakers and video to some display.
+
+### Proxy album art for media browser
+
+Optional. Some media players can serve album artwork or other images on an internal network accessible to Home Assistant that is inaccessible from an external url.
```

Comment:
```suggestion
Optional. If your media player is only accessible from the internal network, it will need to proxy the album art via Home Assistant to be able to work while away from home.
```